contracts/shared/ForwarderXReceiver.sol

```diff
@@ -28,22 +31,47 @@ abstract contract ForwarderXReceiver {
         bytes calldata _callData
     ) external payable onlyConnext {
         // Decode calldata
-        (address fallbackAddress, bytes memory data) = abi
-            .decode(_callData, (address, bytes));
+        (address fallbackAddress, bytes memory data) = abi.decode(
+            _callData,
+            (address, bytes)
+        );
 
         require(fallbackAddress != address(0), "!invalidFallback");
```

Comment:
what happens to funds here? why is it any different than not performing this check?

Reply:
True, removed.